src/components/Welcome/Welcome.jsx

```diff
@@ -1,6 +1,14 @@
 // import the React library here
 // import a css file containig article styles
+import React from 'react';
+import './Welcome.css';
 
 // Create a Welcome function returning the HTML of welcome block
+const Welcome = () => (
+  <section className="welcome">
+    <span className="welcome__text">Sticky Header!</span>
```

Comment:
better to move text on the new line
```suggestion
    <span className="welcome__text">
      Sticky Header!
    </span>
```